### ISSUE TITLE
NewickIO get_comment typo coment

### ISSUE THREAD

**Issue by ktmeaton** [biopython/biopython#3012]
### Setup

I am reporting a bug with Biopython 1.77, Python 3.83, and Ubuntu 18.04 as follows:

The function [Phylo.NewickIO._get_comment](https://github.com/biopython/biopython/blob/master/Bio/Phylo/NewickIO.py#L85) appears to contain a typo in which 'comment' is mispelled as 'coment'.

```python
def _get_comment(clade):
    try:
        comment = clade.coment
```

### Expected behaviour

Calls to this function seem to be unable to parse comments from an input tree (ex. Nexus format), and output tree files have all comments missing. For example when using:

```
Phylo.write(tree, handle, 'nexus')
```

Reverting to Biopython 1.76 restores Nexus comment parsing and output.

**Comment by peterjc** [comment]
You're right - a mistake introduced in d1d3c0d6ab33de12057201e09eb48bdb1964521a as part of Python 2/3 work.

Would you like to submit a pull request fixing this?

Also, if you can add a test of this, even better 👍

**Comment by ktmeaton** [comment]
Thanks for confirming! I'll read the contributor's guide for making a pull request and adding to the test suite.
